app/src/components/governance/LiProposal.vue

```diff
@@ -15,7 +15,7 @@
       <h3 class="li-proposal-title">
         {{ proposal.title }}
       </h3>
-      <p class="li-proposal-description">
+      <p v-if="proposal.description" class="li-proposal-description">
```

Comment:
could there actually be no description?

Reply:
yes, Treasury proposals don't have them. But true, I will just add a template, like the original idea was

Reply:
OK, now treasury also has a description coming from chain

Reply:
from chain or from db?

Reply:
Chain. We are not storing anything proposal-related yet, at least